### DISCUSSION TITLE
Are you planning to develop Basilisk/Pale Moon compatible version?

### DISCUSSION THREAD

**Discussion by Aleksandra0056** [black7375/Firefox-UI-Fix#622] [Q&A] [answered]
Are you planning to develop Firefox UI Fix compatible with Basilisk/Pale Moon? And if no, then please tell why.

https://www.basilisk-browser.org/
https://www.palemoon.org/

**Comment by black7375** [comment, marked as answer]
To support the above browsers, we need to write all the code.
This is a completely new project.

**Reply by Aleksandra0056** [reply]
So, that would be difficult?

**Reply by black7375** [reply]
Yes, I do not plan to support.
And I also recommend using the latest browsers because of security and performance problems.
